examples/Exploring-different-models.ipynb

```diff
@@ -0,0 +1,802 @@
+{
```

Comment:
<p>If you're open to suggestion, please consider repeating the function name.</p><p> </p><p>"We use a utility function, <code>workflow_fit_transform</code> to perform workflow transformation. The function enables us to focus on defining, training, and evaluating models." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Thanks. I will rephrase. </p>